shards/core/runtime.cpp

```diff
@@ -571,13 +572,9 @@ SHWireState suspend(SHContext *context, double seconds) {
     context->next = SHClock::now().time_since_epoch() + SHDuration(seconds);
   }
 
-#ifndef __EMSCRIPTEN__
-  context->continuation = context->continuation.resume();
-#else
-  context->continuation->yield();
-#endif
-
-  // RESUMING
+  SH_CORO_SUSPENDED(context->currentWire());
+  coroutineSuspend(*context->continuation);
+  SH_CORO_RESUMED(context->currentWire());
```

Comment:
This is also weird

Reply:
Just for debug thread names, the suspend is the same as previous